Source/Core/loadImageFromTypedArray.js

```diff
@@ -12,6 +12,10 @@ function loadImageFromTypedArray(options) {
   var format = options.format;
   var request = options.request;
   var flipY = defaultValue(options.flipY, false);
+  var skipColorSpaceConversion = defaultValue(
+    options.skipColorSpaceConversion,
```

Comment:
Tests should be added for this new option. Does the `ImageBitmap` store the `colorSpaceConversion` it was created with? Hopefully, because that will make it pretty straightforward to test.

Reply:
Same with the `Resource` functions.